### PR TITLE
chore/retry transaction button

### LINKED ISSUE #25: Implementar "Retry transaction"  en Tamagotchi

### PR DESCRIPTION
**Pull request by IsmaelPSY**
- feat: upgrade pyhton version for commit
- chore: add retry transaction button

### CODE REVIEW

File: .pre-commit-config.yaml
Comment (utaladriz) [inline .pre-commit-config.yaml:1]: Did you check if python 3.10 is available in fly ?
I think we are deploying tamagotchi on fly

Comment (IsmaelPSY) [reply .pre-commit-config.yaml:1]: this [guide](https://devguide.python.org/versions/#supported-versions) is from fly.io docs, says 3.10 is supported

Comment (IsmaelPSY) [reply .pre-commit-config.yaml:1]: So, changes are ok? :D

Comment (a-mpch) [reply .pre-commit-config.yaml:1]: we are deploying to python 3.10.
Pre-commit only runs on local. **BUT**, if we doing this change we should change poetry version aswell

Comment (a-mpch) [reply .pre-commit-config.yaml:1]: `pyproject.toml` is using 3.9

Comment (IsmaelPSY) [reply .pre-commit-config.yaml:1]: got it.

Comment (utaladriz) [reply .pre-commit-config.yaml:1]: Will you downgrade to 3.9 @IsmaelPSY  or upgrade pyproject.toml  to 3.10 ?

**Review by utaladriz — APPROVED** [review]

**Comment by IsmaelPSY** [comment]
@utaladriz I will upgrade pyproject.toml to 3.10

**Pull request merged**